frontend/src/manage.tsx

```diff
@@ -120,6 +125,20 @@ export function Manage() {
     setSelected(targetOption)
   }
 
+  const filtered = (options: OptionType[]) => {
+    if (showInactive || topic == 'user') {
+      return options;
+    }
+    const genebank = genebanks.find(g => g.name == topic)
+    if (!genebank) {
+      return options;
+    }
+    return options.filter(option => {
+      const herd = genebank.herds.find(h => h.herd == option.value)
```

Comment:
O(n²), how many herds do we have? If they're too many this will become sluggish. But say <100 should be no problem.